burst/Controllers/ApiController.cs

```diff
@@ -4,26 +4,49 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Hosting;
 
 namespace Burst.Controllers
 {
+    using Burst.K8sApi;
+    using System.Threading;
+
     [ApiController]
     [Route("/burstmetrics")]

```

Comment:
general convention is for [Controller] to be the first part of the path - rename class to BurstMetricsController?